docs/_documentations/metrics-dashboard.md

```diff
@@ -75,18 +75,23 @@ The **Profiling** tab is available only in Node.js projects. This tab pulls info
 
 ## Code highlighting for profiling data
 
-Code highlighting for profiling data highlights the most heavily used functions based on profiling data gathered through [load testing](performance.html). Support for code highlighting is available for Node.js or Java projects that are created through Codewind and then profiled.
+Code highlighting for profiling data highlights the most heavily used functions based on profiling data gathered through [load testing](performance.html).
 
-To enable code highlighting, you must install the appropriate VS Code extension:
+### Prerequisites
+Code highlighting is available for Node.js or Java projects on VS Code that are created through Codewind and then profiled.
+
+To enable code highlighting, install the appropriate VS Code extension:
 - For Node.js projects, install the [Codewind Node Profiler](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-node-profiler).
-- For Java projects, install the [Codewind Java Profiler](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-java-profiler). Java is supported, however, if the Metrics Dashboard does not show profiling data, use the editor plug-in to display it instead. 
-- To execute a load test, please refer to **Running a load test** section of the [Performance Dashboard](performance.html).
+- For Java projects, install the [Codewind Java Profiler](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-java-profiler). Java is supported, however, if the Metrics Dashboard does not show profiling data, use the editor plug-in to display it instead.
+
+### Running a load test and load run
```

Comment:
@tetchel This list of bullet points was pretty big, so I divided it into "Prerequisites" and "Running a load test and load run." Do you think that these subheadings are suitable for the info under them? Please let me know if something else would be more clear/accurate. Thank you!

Reply:
Seems OK to me